docker-compose.yml

```diff
@@ -12,4 +12,4 @@ services:
         - "/data/application-data/speech-db/audio:/app/data/audio"
         - "/data/maskwacis-recordings/sessions:/app/data/sessions"
         - "/data/maskwacis-recordings:/data/maskwacis-recordings"
-        - "/data/application-data/speech-db/metadata.csv:/app/private/metadata.csv"
+        - "/data/application-data/speech-db/metadata.csv:/app/private/metadata.csv"
```

Comment:
I expect you didn’t intend to remove the trailing newline here? PyCharm → Preferences → Editor → General → scroll *waayyy* down on the right → “Ensure every saved file ends with a line break”

Reply:
I added a line to this file and pushed it straight to Production, but didn't need that line in this PR, so I removed it and it took the newline with it 🤦 
Good to know PyCharm can stop this from happening though!